### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Sites- Edit_Screen Reader: Narrator is announcing "Read Only" for the "Web Server" and "Web Sites" link

### ISSUE THREAD

**Issue by Shayanti123** [Microsoft/IIS.WebManager#168]
**Environment Details:**
URL:https://manage.iis.net/webserver/certificates
Browser Details: Microsoft Edge 42.17134.1.0 
Screen Reader: Narrator
**MAS ID Violated:** MAS1.3.1

**Repro Steps:**
1) Download and install "Microsoft IIS Administration" application.
2) Enable Internet information Services.
3) Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4) Tab till "Web Sites" in the left menu section and Press Enter.
5) Tab till "More" commands button of the existing website inside the grid and Press Enter.
6) Tab till "Edit" in the expanded menu and Press Enter.
7) Start Screen Reader (Narrator/NVDA).
9) Verify screen reader is reading all the contents, controls, links, images properly throughout the process.

**Actual Result:**
Narrator is "Read Only" for the "Web Server" and "Web Sites" link but on pressing Enter user is able to access the links 

**Expected Result:**
Narrator should not read the links as Read Only as the links are accessible and user is able to view the page

**User Impact:** 
This may create confusion to the screen reader users if the screen reader announces Read Only for interactive controls

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachments:**
[Bug_168_Narrator is reading Read only for the links.pptx](https://github.com/Microsoft/IIS.WebManager/files/2331700/Bug_168_Narrator.is.reading.Read.only.for.the.links.pptx)

**Comment by jhkimnew** [comment]
This is not reproducible in the latest build.

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ScreenReader; #MAS1.3.1; #C+E-H; #.NETWebsites; #Edge; #Narrator

**Comment by Yashi1201** [comment]
**Github tags:** #A11ySev2;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
